src/components/Dropdown/DropdownMenu.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+interface DropdownMenuProps {
+  children: React.ReactNode;
+  /**
+   * Align the menu to the left or right edge of its toggle.
+   */
+  align?: 'left' | 'right';
```

Comment:
since `'left'` and `'right'` are used in a bunch of places could these values be turned into an enum instead, that gets referenced in all those places?